dags/inference/maxtext_inference_microbenchmark.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A DAG to run MaxText inference microbenchmarks with nightly version."""
+
+import datetime
+import pytz
+import itertools
+import numpy
+from airflow import models
+from dags.vm_resource import TpuVersion, Zone, Project, V5_NETWORKS, V5E_SUBNETWORKS, V5P_SUBNETWORKS, RuntimeVersion
+from dags.inference.configs import maxtext_inference_microbenchmark_gce_config
+from dags.multipod.configs.common import SetupMode
+
+LLAMA2_7B = "llama2-7b"
+LLAMA2_13B = "llama2-13b"
+
+W_BF16_KV_BF16 = "w-b16-kv-b16"
+W_BF16_KV_INT8 = "w-b16-kv-i8"
+W_INT8_KV_INT8 = "w-i8-kv-i8"
+W_INT8_KV_BF16 = "w-i8-kv-b16"
```

Comment:
I think W_INT8_KV_BF16 = "w-i8-kv-b16" and W_BF16_KV_INT8 = "w-b16-kv-i8" are not of interest, we can delete these two

Reply:
Done.